.github/workflows/codingStandards.yml

```diff
@@ -7,32 +7,79 @@ on:
   pull_request: { }
 
 jobs:
-  phpstan:
-    name: PHPStan
+  phpstan7:
+    name: PHPStan on PHP 7.4
     runs-on: ubuntu-latest
     steps:
-      - name: Set default PHP7.4
+      - name: Set default php7.4
         run: sudo update-alternatives --set php /usr/bin/php7.4
 
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Clear composer cache
+        run: composer clearcache
+
       - name: Install dependencies
         run: composer install --prefer-dist --no-progress
 
       - name: PHP Stan
         run: vendor/bin/phpstan analyse -c phpstan.neon
 
-  phpcs:
-    name: PHPCS
+  phpcs7:
+    name: PHPCS on PHP 7.4
     runs-on: ubuntu-latest
     steps:
-      - name: Set default PHP7.4
+      - name: Set default php7.4
         run: sudo update-alternatives --set php /usr/bin/php7.4
 
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Clear composer cache
+        run: composer clearcache
+
+      - name: Install dependencies
+        run: composer install --prefer-dist --no-progress
+
+      - name: PHPCS
+        uses: chekalsky/phpcs-action@v1
+        with:
+          phpcs_bin_path: './vendor/bin/phpcs'
+          enable_warnings: true
+
+  phpstan8:
+    name: PHPStan on PHP 8.0
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set default php8.0
+        run: sudo update-alternatives --set php /usr/bin/php8.0
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Clear composer cache
+        run: composer clearcache
+
+      - name: Install dependencies
+        run: composer install --prefer-dist --no-progress
+
+      - name: PHP Stan
+        run: vendor/bin/phpstan analyse -c phpstan.neon
+
+  phpcs8:
+    name: PHPCS on PHP 8.0
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set default php8.0
+        run: sudo update-alternatives --set php /usr/bin/php8.0
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Clear composer cache
```

Comment:
are these required? seems like they are not doing anything really 

![Screenshot 2022-06-03 at 09 44 49](https://user-images.githubusercontent.com/16667281/171811024-ca5974eb-d87f-4662-bbb4-9653aff8c05c.png)